RELEASES.md

```diff
@@ -1,4 +1,15 @@
 # Unreleased
+## Security
```

Comment:
I decided to class this as a "Security" bug because that will cause people to pay attention to it. I'm also inclined to add a security advisory to https://github.com/RustSec/advisory-db to alert other downstream users.

Reply:
Yeah an advisory will push out more updates